src/pseducode.txt

```diff
@@ -0,0 +1,14 @@
+def resolve_uri(path):
+
+
+    if os.path.isdir(path):
+        return html list as body
+        response_ok()
+    elif os.path.isfile(path):
+        return contents as body
+        response_ok()
+    else
+        raise 404 error
+
+
+
```

Comment:
try not to leave a file like this hanging around in a src directory.  It's useful to have a scratch pad, and even useful to keep it in your repo, but i'd suggest that adding a top-level directory called something like "docs" would be a better choice than having it intertwined with all the other actual source code.